controllers/articles.controller.js

```diff
@@ -1,12 +1,23 @@
-const { selectArticleById } = require("../modules/articles.modules")
+const { selectArticleById, selectArticles } = require("../modules/articles.modules")
 
 exports.getArticle = (req, res, next) => {
```

Comment:
We have over complicated this by combining getAllArticles and get ArticleById - we want these to be separate ideally, as it makes our code more organised and easy to read.